Tools/scripts/extract_features.py

```diff
@@ -133,6 +133,7 @@ def __init__(self, filename):
             ('GPS_MOVING_BASELINE', r'AP_GPS_Backend::calculate_moving_base_yaw\b',),
 
             ('HAL_WITH_DSP', r'AP_HAL::DSP::find_peaks\b',),
```

Comment:
does not this need to be removed also? its causing build errors on the wiki:
....
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
feature HAL_WITH_DSP not in build_options.py
......

Reply:
> does not this need to be removed also? its causing build errors on the wiki: .... feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py feature HAL_WITH_DSP not in build_options.py ......

https://github.com/ArduPilot/ardupilot_wiki/pull/4703